irods/connection.py

```diff
@@ -694,6 +694,7 @@ def _login_native(self, password=None):
         )
         self.send(pwd_request)
         self.recv()
+        logger.info("Native authorization validated (in legacy auth).")
```

Comment:
This message seems `debug`-y to me... I see that we're looking for it in the new test, but I'm wondering whether 4.2 users want to see this in the log every time authentication happens?

Also, prefer the term "authentication" but I guess our authentication is also a form of authorization, technically, so it's not wrong. :)

Reply:
There are ancient commits that use "authorization". I balked at this, too.  Should I change the old ones to "authentication"?

Reply:
W.r.t. the level of INFO - no-one would see this, actually, unless they decreased the threshold level of logging from WARNING to INFO.

Reply:
Oh, very good to know. This is fine as-is, then. Resolving.

Reply:
@alanking - Apparently WARNING[30=DEFAULT] > INFO[20] > DEBUG[10]. 
Is that different from the iRODS server logging?

Reply:
The iRODS server's log categories default to "info". Whatever the default is for Python is fine for PRC.